README.md

```diff
@@ -1,29 +1,57 @@
-# The Fermyon developer website
```

Comment:
Is the change here intended?
(this seems to be the readme from the Spin project)

Reply:
This appears resolved now, only one file changed as part of this PR.